.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: C/C++ CI
+
+on:
+  push:
+      branches: [ master ]
+  pull_request:
+      branches: [ master ]
+
+jobs:
+  build-and-test:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@v2
+      - name: install thirdparties
+      run: |
+        sudo apt install -y cmake 
+        sudo apt install -y libboost-filesystem-dev libboost-test-dev libboost-date-time-dev libboost-program-options-dev
+        sudo apt install -y libhdf5-dev
+        sudo apt install -y libeigen3-dev
+        sudo apt install -y libqt4-opengl-dev
+        mkdir -p deps
+        docker pull itisfoundation/iseg-ubuntu-vtk:8.2
```

Comment:
you might be tempted to use some matrix at some point in case you might need to test other versions.
see [https://github.com/ITISFoundation/osparc-simcore/blob/79a24d1374557496e1bc21cf9cd8147b62bf6d7d/.github/workflows/ci-testing-deploy.yml](https://github.com/ITISFoundation/osparc-simcore/blob/79a24d1374557496e1bc21cf9cd8147b62bf6d7d/.github/workflows/ci-testing-deploy.yml) for an example. Lines 41-48. not sure it applies here.

Reply:
thanks. I will have look. ubuntu 20.04 doesn't have qt4 anymore, so I will need to get rid of that ancient dependency at last...